draft-ietf-rtcweb-jsep.xml

```diff
@@ -1990,8 +1990,13 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               the dummy value "9 IN IP4 0.0.0.0", because no candidates
               have yet been gathered.</t>
 
-              <t>A direction attribute which is the same as that of the
-              associated transceiver.</t>
+              <t>A direction attribute, determined by intersecting the
+              direction of the associated RtpTransceiver with the 
+              rules regarding the offer direction specified
+              in <xref target="RFC3264" />, Section 6.1. For example, 
```

Comment:
This prose feels a bit clunky though I'm not entirely sure what the best prose would be.

Perhaps "determined by applying the rules for offer direction specified in [...] and then intersecting with the direction of the associated RtpTransceiver"